src/NzbDrone.Core/Datastore/Migration/066_add_tags.cs

```diff
@@ -16,6 +16,9 @@ protected override void MainDbUpgrade()
 
             Alter.Table("Notifications")
                  .AddColumn("Tags").AsString().Nullable();
+
+            Execute.Sql("UPDATE Series SET Tags = '[]'");
+            Execute.Sql("UPDATE Notifications SET Tags = '[]'");
```

Comment:
Its other migrations that would be the issue, if the user is coming from a really old DB version the migrations don't null check and it fails.

This should have been done originally, but this will help anyone else that upgrades from an old version.